inspirehep/base/searchext/mappings/hep.json

```diff
@@ -57,15 +57,32 @@
                         }
                     }
                 },
+                "abstracts": {
+                    "type": "object",
+                    "properties": {
+                        "value": {
+                            "type": "string",
+                            "copy_to": ["abstract", "global_fulltext"]
+                        }
```

Comment:
Maybe via something like in https://www.elastic.co/guide/en/elasticsearch/reference/2.1/analysis-custom-analyzer.html It's probably worth a dedicated github issue.